packages/compass-e2e-tests/tests/collection-import.test.ts

```diff
@@ -509,6 +509,129 @@ describe('Collection import', function () {
     await toastElement.waitForDisplayed({ reverse: true });
   });
 
+  context('with validation', function () {
+    beforeEach(async function () {
+      const FAILING_VALIDATOR =
+        '{ $jsonSchema: { bsonType: "object", required: [ "abcdefgh" ] } }';
+      await browser.setValidation({
+        connectionName: DEFAULT_CONNECTION_NAME_1,
+        database: 'test',
+        collection: 'extended-json-file',
+        validator: FAILING_VALIDATOR,
+      });
+    });
+
+    afterEach(async function () {
+      await browser.navigateWithinCurrentCollectionTabs('Validation');
+      await browser.setValidationWithinValidationTab('{}');
```

Comment:
Would it make sense to instead run this resetting validation operation against the db directly and not use the UI for the cleanup? I'm thinking if the validation setting does fail this won't clean up and then more tests may start failing.
Or maybe the collection gets dropped altogether and we don't need this?

Reply:
Part of me things it's better to do both directly that way we don't have to change things around when we change things that aren't related to this test. That way there are less cross dependencies and we're only focusing on the test target. That being said I also think fine it's fine as is, more of a nit. It might also speed up tests to avoid extra UI things.